drivers/display/display_st7789v.c

```diff
@@ -352,6 +352,11 @@ static int st7789v_init(const struct device *dev)
 #if DT_INST_SPI_DEV_HAS_CS_GPIOS(0)
 	data->cs_ctrl.gpio_dev = device_get_binding(
 			DT_INST_SPI_DEV_CS_GPIOS_LABEL(0));
+	if (data->cs_ctrl.gpio_dev == NULL) {
```

Comment:
Is there a reason why this code was added? I would assume that this check is not mandatory as the device tree will guarantee that the device is present or the code will not be compiled in (ifdef at line 352).